### PR TITLE
fix(editor): Fix Nodeview.v2 reinitialise based on route changes

### PR DESCRIPTION
**Pull request by r00gm**
## Related Linear tickets, Github issues, and Community forum posts

[PAY-2336](https://linear.app/n8n/issue/PAY-2336/add-workflow-button-not-working)

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/editor-ui/src/views/NodeView.v2.vue [hunk 1]
```diff
@@ -300,6 +300,7 @@ async function initializeRoute() {
 	}
 
 	const isAlreadyInitialized =
+		route.params.name === initializedWorkflowId.value &&
 		initializedWorkflowId.value &&
 		[NEW_WORKFLOW_ID, workflowId.value].includes(initializedWorkflowId.value);
```
Comment (MarcL) [inline packages/editor-ui/src/views/NodeView.v2.vue:306]: Is there a test we can add to check this change?

Comment (MarcL) [reply packages/editor-ui/src/views/NodeView.v2.vue]: As there are no tests right now, we can merge this but look at testing as the next steps.

**Review by alexgrozav — APPROVED** [review]
Works as expected. Thank you!

**Comment by janober** [comment]
Got released with [`n8n@1.70.3`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.70.3)

**Pull request merged**